File: src/main/java/com/gioia/radiogio/data/repositories/RadioStationRepository.java
package com.gioia.radiogio.data.repositories;

import com.gioia.radiogio.data.domains.RadioStation;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface RadioStationRepository extends JpaRepository<RadioStation, String> {
}

File: src/main/kotlin/com/gioia/radio/data/repositories/RadioStationRepository.kt
package com.gioia.radio.data.repositories

import com.gioia.radio.data.domains.RadioStation

interface RadioStationRepository {
    fun removeAll()
    fun saveAll(radioStation: List<RadioStation>)
    fun createIndexes()
    fun getInitialRadioStations(): List<RadioStation>
    fun findByCountryNameLike(countryCode: String): List<RadioStation>
    fun findByRadioNameLike(radioName: String): List<RadioStation>
}